src/app/frontend/replicasetdetail/replicaset_service.js

```diff
@@ -64,4 +66,41 @@ export class DeleteReplicaSetService {
 
     return deferred.promise;
   }
+
+  /**
+   * Opens an update replica set dialog. Returns a promise that is resolved/rejected when user wants
+   * to update the replicas. Nothing happens when user clicks cancel on the dialog.
+   *
+   * @param {!backendApi.ReplicaSetDetail} replicaSetDetail
+   * @returns {!angular.$q.Promise}
+   */
+  showUpdateReplicasDialog(replicaSetDetail) {
```

Comment:
Totally agree. I will make it simpler. On details page it had more sense to leave it like that.